1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,27 @@
 function readingList(books) {
```

Comment:
This is absolutely brillant. It ticks all the boxes. const used where you need it, triple equals, sensible varible names. The only complaint is the formatting, same on the other files. Now you're getting to the proper projects it's a bit more of a deal. Have I reminded you about  Prettier (see https://docs.codeyourfuture.io/course-processes/before-the-course/getting-setup) and the code style guide (https://syllabus.codeyourfuture.io/guides/code-style-guide/)?